### PR TITLE
feat: voting v2 minor refactoring

### PR DESCRIPTION
**Pull request by md0x**
**Motivation**

Minor refactoring proposal and missing natspec

### CODE REVIEW

File: packages/core/contracts/data-verification-mechanism/implementation/VotingV2.sol [hunk 1]
```diff
@@ -36,7 +36,7 @@ contract VotingV2 is Staker, OracleInterface, OracleAncillaryInterface, OracleGo
         uint64 time; // Timestamp used when evaluating the request.
         uint32 rollCount; // The number of rounds that a price request has rolled. Informs if a request can be deleted.
         bytes32 identifier; // Identifier that defines how the voters should resolve the request.
-        mapping(uint256 => VoteInstance) voteInstances; // A map containing all votes for this price in various rounds.
+        mapping(uint32 => VoteInstance) voteInstances; // A map containing all votes for this price in various rounds.
```
Comment (md0x) [inline packages/core/contracts/data-verification-mechanism/implementation/VotingV2.sol:39]: We can use `uint32` the same type used for [roundIds](https://github.com/UMAprotocol/protocol/blob/master/packages/core/contracts/data-verification-mechanism/implementation/VotingV2.sol#L133)

File: packages/core/contracts/data-verification-mechanism/implementation/VotingV2.sol [hunk 2]
```diff
@@ -54,7 +54,7 @@ contract VotingV2 is Staker, OracleInterface, OracleAncillaryInterface, OracleGo
         uint128 minParticipationRequirement; // Minimum staked tokens that must vote to resolve a request.
         uint128 minAgreementRequirement; // Minimum staked tokens that must agree on an outcome to resolve a request.
         uint128 cumulativeStakeAtRound; // Total staked tokens at the start of the round.
-        uint32 numberOfRequestsToVoteOnInThisRound; // The number of requests that have been voted on in this round.
+        uint32 requestsNumber; // The number of requests that have been voted on in this round.
```
Comment (md0x) [inline packages/core/contracts/data-verification-mechanism/implementation/VotingV2.sol:57]: `numberOfRequestsToVoteOnInThisRound` seemed, redundant, particularly the "InThisRound" as this is already a variable of  `Round`.

Comment (Reinis-FRP) [reply packages/core/contracts/data-verification-mechanism/implementation/VotingV2.sol:57]: This seems bit too generic. Maybe `numberOfRequestsToVote`.

Also the comment should better state "The number of requests that have to be voted on in this round."

Comment (md0x) [reply packages/core/contracts/data-verification-mechanism/implementation/VotingV2.sol]: Fixed in [this commit](https://github.com/UMAprotocol/protocol/pull/4385/commits/fbee9463f95e458044ef7a08c0dc4045790cebef)

**Review by Reinis-FRP — APPROVED** [review]

**Review by chrismaree — APPROVED** [review]

**Review by mrice32 — APPROVED** [review]
LGTM!

**Pull request merged**